[2022-12-07T20:54:14.944+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag.create_prediction_table manual__2022-12-07T20:53:53.396680+00:00 [queued]>
[2022-12-07T20:54:14.966+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag.create_prediction_table manual__2022-12-07T20:53:53.396680+00:00 [queued]>
[2022-12-07T20:54:14.967+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-12-07T20:54:14.968+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-12-07T20:54:14.968+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-12-07T20:54:14.999+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_prediction_table> on 2022-12-07 20:53:53.396680+00:00
[2022-12-07T20:54:15.009+0000] {standard_task_runner.py:55} INFO - Started process 9833 to run task
[2022-12-07T20:54:15.015+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag', 'create_prediction_table', 'manual__2022-12-07T20:53:53.396680+00:00', '--job-id', '1946', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp2qs3_u4a']
[2022-12-07T20:54:15.019+0000] {standard_task_runner.py:83} INFO - Job 1946: Subtask create_prediction_table
[2022-12-07T20:54:15.112+0000] {task_command.py:376} INFO - Running <TaskInstance: dag.create_prediction_table manual__2022-12-07T20:53:53.396680+00:00 [running]> on host a9d6893ff63e
[2022-12-07T20:54:15.287+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=lofitravel
AIRFLOW_CTX_DAG_ID=dag
AIRFLOW_CTX_TASK_ID=create_prediction_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-07T20:53:53.396680+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-07T20:53:53.396680+00:00
[2022-12-07T20:54:15.302+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-12-07T20:54:15.320+0000] {sql.py:315} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS prediction_table(
            date TIMESTAMP NOT NULL,
            ticker_from VARCHAR(3) NOT NULL,
            ticker_to VARCHAR(3) NOT NULL,
            amount FLOAT NOT NULL,
        PRIMARY KEY (date, ticker_from, ticker_to));
        , parameters: None
[2022-12-07T20:54:15.322+0000] {postgres.py:96} INFO - NOTICE:  relation "prediction_table" already exists, skipping

[2022-12-07T20:54:15.339+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag, task_id=create_prediction_table, execution_date=20221207T205353, start_date=20221207T205414, end_date=20221207T205415
[2022-12-07T20:54:15.390+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-07T20:54:15.428+0000] {taskinstance.py:2623} INFO - 1 downstream tasks scheduled from follow-on schedule check
